.gitignore

```diff
@@ -0,0 +1 @@
+test.md
```

Comment:
- [ ] Kindly add node_modules to the .gitignore, it's good practice since you can end up accidentally pushing it when setting up the local Linter.